cmd/consensus/main.go

```diff
@@ -225,6 +225,8 @@ func main() {
 				filter.HasRole(flow.RoleExecution),
 				func() flow.Entity { return &flow.ExecutionReceipt{} },
 				// requester.WithBatchThreshold(100),
+				requester.WithRetryInitial(2*time.Second),
+				requester.WithRetryMaximum(30*time.Second),
```

Comment:
the default for the `RetryMaximum` was 2min, which is way too slow as it directly impacts time to seal , which is critical for NBA